roles/fedora/templates/tomcat7_leveldb.j2

```diff
@@ -1,5 +1,5 @@
-# A backup of the original file with addition options is at /etc/default/tomcat7.bak
-TOMCAT7_USER=tomcat7
```

Comment:
Would it be useful to leave a comment showing what the final generated line will look like?

```
# This template will construct settings that look something like
# TOMCAT7_USER=tomcat7
# TOMCAT7_GROUP=tomcat7
```

If I hadn't seen the diff, I'd have a really hard time coming to this code and making sense of it.  At a minimum, I'd have to find where servlet is defined in defaults or elsewhere to have a sense of what's happening here.